docs/howtos/audit-scanner.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_label: "Audit Scanner Installation"
+title: "Audit Scanner Installation"
+---
+
+# Audit Scanner installation
+
+Starting from version `v1.7.0`, Kubewarden introduces a new component called
+the Audit Scanner. This scanner evaluates whether the resources running in the
+cluster comply with the deployed policies. It is particularly useful when
+policies are deployed after the resources. The Audit Scanner allows operators
+to detect resources deployed before the policies that violate the validations
+performed by the policies.
+
+## Installation
+
+To install and use the Kubewarden Audit Scanner, please follow these steps:
+
+1. Install the `kubewarden-controller` Helm chart, ensuring you have version
+   `v1.7.0` or higher. Remember to enable the audit scanner component.
+
+   ```console
+   helm install kubewarden-crds kubewarden/kubewarden-crds
+   helm install --set auditScanner.enable=true kubewarden-controller kubewarden/kubewarden-controller
+   ```
+
+   For more information about the installation of Kubewarden see the [Quick Start guide](../quick-start.md)
+
+:::caution
+   The PolicyReport Custom Resource Definitions must be available to store the
+   policy report results. Therefore, if the `kubewarden-crds` is installed with
+   the `installPolicyReportCRDs` value set to `false`, the cluster operator should
+   install the CRD manually. Note that, the `kubewarden-crds` installs the CRDs
+   by default.
+
+   See more info about the CRDs at the [policy work group
+   repository](https://github.com/kubernetes-sigs/wg-policy-prototypes)
+:::
+
+By default, the Audit Scanner is implemented as a cron job that will be
+triggered every 60 minutes. You can adjust this and other audit scanner
+configuration changing other chart values. Check the
+[values.yaml](https://github.com/kubewarden/helm-charts/blob/main/charts/kubewarden-controller/values.yaml)
```

Comment:
I would remove the line "Check the file for more options" as it is understood from the previous line.